scripts/release/asset_publish.py

```diff
@@ -63,22 +65,30 @@ def preprocess_test_files(test_jobs, asset_ids: dict):
                           sort_keys=False)
 
 
-def dump_model_spec(model, spec_file) -> bool:
-    """Update the yaml file after getting the model has been prepared."""
+def update_spec(asset: Any, spec_file: Path) -> bool:
+    """Update the yaml spec file with updated properties in asset.
+
+    :param asset: Asset loaded using load_*(component, environemnt, model) method.
+    :type asset: Any
+    :param spec_file: path to model spec file
+    :type spec_file: Path
+    :return: True if spec was successfully updated
+    :rtype: bool
+    """
     try:
-        model_dict = json.loads(json.dumps(model._to_dict()))
-        util.dump_yaml(model_dict, spec_file)
+        asset_dict = json.loads(json.dumps(asset._to_dict()))
+        util.dump_yaml(asset_dict, spec_file)
         return True
     except Exception as e:
-        logger.log_error(f"Failed to update model spec => {str(e)}")
+        logger.log_error(f"Failed to update spec => {str(e)}")
```

Comment:
For f-strings, `str()` is implicit.
```suggestion
        logger.log_error(f"Failed to update spec: {e}")
```

Reply:
Done, thank you.